From 9c9a98398b49203fa8f752db306c3996d5a5f1f4 Mon Sep 17 00:00:00 2001
From: Yijie Yang <quic_yijiyang@quicinc.com>
Date: Fri, 26 Jul 2024 09:14:26 +0800
Subject: [PATCH 22/30] phy: qcom: add support for SGMII SerDes PHY on qcs8300
 platform

Add new compatible to support SGMII/SerDes PHY on qcs8300.

Signed-off-by: Yijie Yang <quic_yijiyang@quicinc.com>
---
 drivers/phy/qualcomm/phy-qcom-sgmii-eth.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/phy/qualcomm/phy-qcom-sgmii-eth.c b/drivers/phy/qualcomm/phy-qcom-sgmii-eth.c
index 5b1c82459c12..b52bff53267f 100644
--- a/drivers/phy/qualcomm/phy-qcom-sgmii-eth.c
+++ b/drivers/phy/qualcomm/phy-qcom-sgmii-eth.c
@@ -353,6 +353,7 @@ static int qcom_dwmac_sgmii_phy_probe(struct platform_device *pdev)
 }
 
 static const struct of_device_id qcom_dwmac_sgmii_phy_of_match[] = {
+	{ .compatible = "qcom,qcs8300-dwmac-sgmii-phy" },
 	{ .compatible = "qcom,sa8775p-dwmac-sgmii-phy" },
 	{ },
 };
-- 
2.34.1

